### ISSUE TITLE
Error in default Svelte sandbox

### ISSUE THREAD

**Issue by plmrry** [codesandbox/codesandbox-client#4871]
# 🐛 bug report

## Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

## Description of the problem

After creating a Svelte sandbox and editing `App.svelte`, the console logs:

```
Error when converting '/node_modules/svelte/internal/index.mjs' esmodule to commonjs: [1253:376]: Unterminated regular expression 
```

### To Reproduce
1. Create a new Svelte sandbox.
2. Edit `App.svelte`
3. Open the console

### Your Environment

| Software         | Name/Version |
| ---------------- | ------------ |
| Browser          | Chrome 85.0.4183.102
| Operating System | macOS 10.15.6

**Comment by SaraVieira** [comment]
Hey!

Yeah, I can see that and we will get on to it, I can't really promise in terms of time or priority since it doesn't break any code but I do get it's quite confusing

Marking it as bug

Thank you

**Comment by jmcantrell** [comment]
Any update on this? It's annoying seeing that warning in the console constantly.

**Comment by imrishabh18** [comment]
@SaraVieira Are you working on this issue? Or can I look into it

**Comment by SaraVieira** [comment]
I'm not sorry :/

Feel free to pick it up :)

**Comment by lbogdan** [comment]
Hey @imrishabh18 ,

Did you manage to take a look at this? Let us know if we can assist in any way.

**Comment by imrishabh18** [comment]
@lbogdan Couldn't do much to resolve this.

**Comment by DeMoorJasper** [comment]
I think I've fixed this as part of #5856 as I can no longer reproduce this and error seems related, feel free to let me know and reopen the issue if it still happens
